hibernate-core/src/main/java/org/hibernate/boot/beanvalidation/TypeSafeActivator.java

```diff
@@ -91,7 +91,9 @@ public static void activate(ActivationContext activationContext) {
 				throw new IntegrationException( "Bean Validation provider was not available, but 'ddl' validation was requested", e );
 			}
 
-			LOG.debug( "Unable to acquire Bean Validation ValidatorFactory, skipping activation" );
+			if (LOG.isDebugEnabled()) {
```

Comment:
I'm not sure these really needed to be protected, but its fine since you did it already